types/delegates/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for delegates 1.0
+// Project: https://github.com/tj/node-delegates
+// Definitions by: xingyuzhe <https://github.com/xingyuzhe>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+interface Delegate {
+    method(name: string): Delegate;
+    access(name: string): Delegate;
+    getter(name: string): Delegate;
+    setter(name: string): Delegate;
+    fluent(name: string): Delegate;
+}
+
+declare function dg(proto: object, target: string): Delegate;
+
+export = dg;
```

Comment:
is this the best representation of the module exported symbol?
It looks like a constructor like syntax (with Function and static methods and prototype methods) with support for creator function, so this import:
```ts
const Delegator = require('delegates');
```
can be called directly or with `new` keyword
```ts
 new Delegator()
Delegator {
  proto: undefined,
  target: undefined,
  methods: [],
  getters: [],
  setters: [],
  fluents: []
}
> Delegator()
Delegator {
  proto: undefined,
  target: undefined,
  methods: [],
  getters: [],
  setters: [],
  fluents: []
}
```
(sorry for crude sample, just to point a problem)

Reply:
I've fixed the representation of the module exported symbol. Please check again.